Makefile

```diff
@@ -16,19 +16,30 @@
 # documented in gen/src/3.6/Doc/Makefile as we're only delegating the
 # real work to the Python Doc Makefile.
 
-CPYTHON_PATH := $(realpath ../cpython/)
-LANGUAGE := fr
-UPSTREAM := https://github.com/python/cpython
-VENV := $(shell pwd)/venv/
-PYTHON := $(shell which python3)
-MODE := html
-BRANCH := 3.8
+# Configuration
+
+# The version of poutils we use (Hardcoding it allows for pip to skip
+# hitting internet if it's already installed).
+POUTILS_VERSION := 0.1.2
```

Comment:
I assume there will be a new version soon: Seluj78/poutils#5 @Seluj78

Reply:
We'll bump it, that's not an issue. If you don't store the version, a `pip install poutils` will reach internet every time we run `make venv` which itself is a phony dependency of most commands. And I don't want this, I want a `make todo` to be fast, not to fetch on pypi every time.

On the other hand, not making `venv` a phony dependency will lead to the previous concensus: never upgrading dependencies, which is also bad.

I think storing the version is a good compromise: everyone gets up-to-date versions, no one hit internet when not needed. We could however store it in a requirements.txt so we can use the dependency bot.

Reply:
Yes, I do understand the purpose, and your solution seems excellent to me if there is auto-bumping by a bot.

Or maybe should I release not so often new functionalities in padpo :wink: 

I'm kidding, but using a bot helps avoiding issues in the future. If there is a requirements.txt, dependabot will make a PR each time a new version of the tools is released (it's free for free software, and has been acquired recently by GitHub, https://github.com/marketplace/dependabot-preview).

Reply:
Moved to a requirements.txt file.